data/sles4sap/b1_log.sh

```diff
@@ -51,11 +63,21 @@ fix_broken_log $second
 sed -i -e '/.*\[Sap Note.*/!d' $fixed
 echo "Sap Notes:"
 while IFS="" read -r line; do
-    if (echo $line | grep -E -v SUCCESS\|INFO); then
+    if (echo $line | grep -q ERROR); then
         failures=$((failures+1))
     fi
+    if (echo $line | grep -q SUCCESS); then
+        success=$((success+1))
+    fi
+    if (echo $line | grep -q INFO); then
+        info=$((info+1))
+    fi
+    if (echo $line | grep -q WARNING); then
+        warnings=$((warnings+1))
+    fi
```

Comment:
Same suggestion here:
```suggestion
    echo $line | grep -q ERROR && failures=$((failures+1))
    echo $line | grep -q SUCCESS && success=$((success+1))
    echo $line | grep -q INFO && info=$((info+1))
    echo $line | grep -q WARNING && warnings=$((warnings+1))
```